src/uv_encoding.c

```diff
@@ -237,7 +237,7 @@ int uvEncodeMessage(const struct raft_message *message,
             return RAFT_MALFORMED;
     };
 
-    header.base = raft_malloc(header.len);
+    header.base = raft_calloc(1, header.len);
```

Comment:
Regarding this one, the situation is similar, there are some cases in which some bytes are not used and the code might net actually set it.

However I would probably prefer to figure out which memory exactly does not get initialized. Most probably one such case is in this line:

https://github.com/atlesn/raft/blob/07b5f3e315c179f7de41c3b519aaacb1563c2f87/src/uv_encoding.c#L340

where 3 bytes are skipped because they are not used. That line should be replaced with something like this instead:

```C
/* Unused */
bytePut8(&cursor, 0);
bytePut8(&cursor, 0);
bytePut8(&cursor, 0);
```

I looked at all the encoding code (the one which populates this `header.base` buffer) and I did not find any other case where memory is not initialized.

So I'd start with the fix above first and see if valgrid is happy. What do you think?